Documentation/Ctrl/Properties/HideAtCopy.rst

```diff
@@ -15,5 +15,6 @@ hideAtCopy
    If set, and the "disabled" field from :ref:`enablecolumns <ctrl-reference-enablecolumns>` is
    specified, then records will be disabled/hidden when they are copied.
 
-   Note it is possible to disable this feature on a page and user or group level using the Page
+   .. seealso::
+   It is possible to disable this feature on a page and user or group level using the Page
```

Comment:
Please indent this paragraph to be displayed in the admonition.

Reply:
Missed that indeed, thanks.